code/datums/looping_sounds/looping_sound.dm

```diff
@@ -48,11 +48,13 @@
 		start()
 
 /datum/looping_sound/Destroy()
+	GLOB.looping_sounds -= src
 	stop()
 	output_atoms = null
 	return ..()
 
 /datum/looping_sound/proc/start(atom/add_thing)
```

Comment:
this proc `start` won't be called multiple times on the same datum will it?

Reply:
no, it starts the loop and then `sound_loop` is called on loop with a timer